### ISSUE TITLE
now it has too much output. 

### ISSUE THREAD

**Issue by leek5pin** [devshawn/ulysses#7]
it's happening before prettifyOutput, and doesn't seem to be related to my handling of slots. I suspect the number of volunteers is somehow relevant.

**Comment by leek5pin** [comment]
Fixed it in a sort of round about way. It looks nice now though. Still can't figure out what was wrong in the first place. 

Will wait to close this until somebody else tries my thing.

**Comment by devshawn** [comment]
It is still not working for me.

Create a time slot for all day (8am to 5:30pm or so) and then make it have around 30 volunteers needed. There are 60 or so volunteers with no conflicts, so this should easily fill this one slot. However, I'm only getting around 12-15 results it seems.

**Comment by leek5pin** [comment]
Yes, me too. By "works" I meant it is not optimal, but it should be a valid output.

**Comment by leek5pin** [comment]
I figured out my error. I had a return two parenthesis too deep. It works
beautifully now. I'll push it up as soon as vinyl unfreezes. I also
accidentally told it to print 300,000 things twice.

**Comment by devshawn** [comment]
This is not fixed. I am getting an array of 25 volunteers for a time slot with 30 people. To replicate, see my original comment.

**Comment by leek5pin** [comment]
That's curious. 

Javascript is doing some weird optimizations behind the scenes. Things are interacting outside of their scope.

**Comment by leek5pin** [comment]
A concat was somehow returning a smaller array than the documentation says is possible, and start seems to be a reserved word for objects? i have no idea... I got it to work for the example you gave me. I'm going to harass it some more, because it's java script.

**Comment by leek5pin** [comment]
Oh, and it's just wonderful that console.log("hello world") changes behavior of code around it.

**Comment by leek5pin** [comment]
I'm giving up for tonight. What I'm about to push seems to work, not taking into account prior commitments.
